Nie-ontologies/Generic-ontologies/publishing.ttl

```diff
@@ -332,7 +332,7 @@ publish:hasPublishingState
 publish:hasPublishing
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/publishing#>;
 	a owl:ObjectProperty;
-	rdfs:label "has publishing"@en, "hat Herausgeben"@de, "ha editoria"@it; #""@fr; ?  
+	rdfs:label "has publishing"@en, "hat Herausgeber"@de, "ha editoria"@it; #""@fr; ?  
```

Comment:
The object is a procedure, not an agent.

Reply:
Ok.

Reply:
Commit 47b31d8